large-file-uploader-app/assets/iframe.html

```diff
@@ -1,6 +1,7 @@
 <html>
 <head>
 	<script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0.24/dist/index.min.css">
```

Comment:
I would move the link above the link tag above the script tag.

Reply:
Absolutely!